### PR TITLE
{bio}[foss/2018b] split-seq v20190717 w/ Python 3.6.6

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/da7a8a0bec2fb8e17916b0cabfab40b5 for a full test report.

File: easybuild/easyconfigs/s/split-seq/split-seq-20191107-add_pkl_barcodes.patch [hunk 1]
```diff
@@ -0,0 +1,17 @@
+--- MANIFEST.in	2019-11-13 15:09:45.174322190 +0100
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/s/split-seq/split-seq-20191107-add_pkl_barcodes.patch]: please add description and author

File: easybuild/easyconfigs/s/split-seq/split-seq-20191107-foss-2018b-Python-3.6.6.eb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+use_pip = True
+download_dep_fail = True
+sanity_pip_check = True
+
+sanity_check_paths = {
+    'files': ['bin/%(name)s'],
+    'dirs': ['lib/python%%(pyshortver)s/site-packages/%s' % name.replace('-', '_')],
+}
+
+modloadmsg = "Reference database located in /databases/bio/%(name)s-%(version)s\n"
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/s/split-seq/split-seq-20191107-foss-2018b-Python-3.6.6.eb]: this is site-specific...

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/7af48276354e955621710de4aa86f2d1 for a full test report.

File: easybuild/easyconfigs/s/split-seq/split-seq-20191107-add_pkl_barcodes.patch (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/s/split-seq/split-seq-20191107-add_pkl_barcodes.patch]: I made that patch to push it upstream, but in hindsight it is overkill for this easyconfig. That fix can be done with a simple `cp` command in `postinstallcmds`. So I updated the easyconfig to not need any patch files.

File: easybuild/easyconfigs/s/split-seq/split-seq-20191107-foss-2018b-Python-3.6.6.eb (hunk 2 above)
Comment (lexming) [reply easybuild/easyconfigs/s/split-seq/split-seq-20191107-foss-2018b-Python-3.6.6.eb]: Absolutely, I updated it to be site agnostic and commented it out as well

**Comment by lexming** [comment]
@migueldiascosta I switched the sources of this easyconfig to the main fork of `split-seq`, as it is actively developed. I'll submit new tests.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1f2f6438117259cddea758230ecf1472 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c2873ce1addbe9f8d2d039e48fe5e887 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/8fcbdab907eb46ea31d4a7c247b7d2a8 for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @lexming!

**Pull request merged**